cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -2,126 +2,21 @@ package cli
 
 import (
 	"fmt"
-	"io"
-	"os"
-	"time"
 
 	"github.com/alecthomas/kong"
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/pkg/errors"
-	"github.com/sirupsen/logrus"
-	"github.com/spiffe/go-spiffe/v2/spiffeid"
 	"github.com/square/spiffe-aws-assume-role/cmd/spiffe-aws-assume-role/cli/mappers"
-	"github.com/square/spiffe-aws-assume-role/pkg/credentials"
-	"github.com/square/spiffe-aws-assume-role/pkg/processcreds"
-	"github.com/square/spiffe-aws-assume-role/pkg/telemetry"
 )
```

Comment:
The huge number of imports was a hint that maybe this file was getting a bit bloated.